### PR TITLE
[New] Add hack-font@3.000 w/ git auto-update

### PR DESCRIPTION
**Pull request by sufuf3**
Pull request for issue: #5598
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/source-foundry/Hack
 * Official website (optional, not the repository): https://sourcefoundry.org/hack/
 * License and its reference: [MIT, BITSTREAM VERA LICENSE](https://github.com/source-foundry/Hack/blob/master/LICENSE.md)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 255
   - Count of stars: 10372
   - Count of forks: 341

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 800 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by sufuf3** [comment]
@chrissimpkins Thank you for your attention. :)
If this PR be merged, I'll send a PR to add CDNJS badge. 
Many thanks 👍

**Comment by sufuf3** [comment]
@cdnjs/intern3 @cdnjs/library-reviewer @cdnjs/active-contributor 
Please help me review this PR.
I use `+(web|webfonts)/**/*` pattern to fetch files of v3.000 under `web` and fetch files of v2.013~v2.020 under `webfonts`.
And I didn't find *.css files in v1.0.0~v2.010 so I didn't manually add them or configure the auto-update config.

Thank you.

File: ajax/libs/hack-font/package.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "hack-font",
+  "filename": "web/hack.min.css",
+  "version": "3.000",
```
Comment (extend1994) [inline ajax/libs/hack-font/package.json]: Seem a little strange here. Normally it's like https://github.com/source-foundry/Hack/blob/master/package.json#L3. Maybe need to ask the author if he can improve this?

**Comment by chrissimpkins** [comment]
Let me know what I can do.

**Comment by sufuf3** [comment]
@chrissimpkins Thanks
our version is depend on [git tag name](https://github.com/source-foundry/Hack/tags)
Would you mind releasing the version name which can be follow [semantic versioning](https://semver.org/spec/v1.0.0.html) in future?
This is good for people to know the version.
If you agree with this method, the bester way is added a new version like `v3.0.1` by you on [repo](https://github.com/source-foundry/Hack/tags)... then I will manually add v3.0.0 and the old versions.
What do you think?

Many thanks.

@extend1994 Not sure is this a good way, if you have any idea, could you join this discussion, please?
Thanks

**Comment by chrissimpkins** [comment]
Unfortunately font versioning doesn't follow SemVer:

https://github.com/openfv/openfv 

Is there another possible approach?

**Comment by sufuf3** [comment]
@chrissimpkins Thanks.
In fact, I think it is not very serious.
Because the lib still can be hosted.
But I'm not sure.
@extend1994 @PeterDaveHello Could you give us some suggestions about the version issue?
Should the lib really need to be followed semantic versioning rule?
Could the lib just follow their version naming?

Many thanks.

**Comment by PeterDaveHello** [comment]
We don't need `semver`, just make sure it's versioning.

**Comment by chrissimpkins** [comment]
We are definitely versioning and we tag each release as well as have GitHub releases for every release.  Our Linux, homebrew, and Chocolatey packages all rely on the current release structure so it would be extremely helpful to avoid changes to the current approach.  Thanks for considering Hack for inclusion here.  Let us know if how/if we can be of additional help!

**Comment by sufuf3** [comment]
Thank you all.
@extend1994 I modified the commit message, please help me review this PR, thanks.

**Pull request merged**